tests/wpunit/Tribe/Tickets/Partials/V2/Components/Icons/ErrorTest.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace Tribe\Tickets\Partials\V2\Components\Icons;
+
+use Codeception\TestCase\WPTestCase;
+use Spatie\Snapshots\MatchesSnapshots;
+use tad\WP\Snapshots\WPHtmlOutputDriver;
+
+class Error extends WPTestCase {
```

Comment:
Thanks for the review!

1. I added the suffix. I didn't feel it was a necessity, but I feel it could be good for consistency.
2. We already have partial testing in the parent folder, that's why I chose this location. 
3. I think `HtmlPartialTestCase` was mostly shaped for views v2 and its structure, and it doesn't work out of the box for this case -> https://github.com/moderntribe/tribe-testing-facilities/blob/master/src/Products/WPBrowser/Views/V2/HtmlPartialTestCase.php#L59